packages/elements/src/internals/machines/sign-in/utils/starting-factors.ts

```diff
@@ -14,10 +14,10 @@ const findFactorForIdentifier = (i: string | null) => (f: SignInFactor) => {
 // The algorithm can be found at
 // https://www.notion.so/clerkdev/Implement-sign-in-alt-methods-e6e60ffb644645b3a0553b50556468ce
 export function determineStartingSignInFactor(
-  firstFactors: SignInFactor[],
+  firstFactors: SignInFirstFactor[],
```

Comment:
`SignInFactor` is `SignInFirstFactor | SignInSecondFactor`. The changes in this file narrow the input and return types